dhcpv6/option_bootfileparam.go

```diff
@@ -0,0 +1,76 @@
+package dhcpv6
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+)
+
+var (
+	// bootpEndian is the endianness used to parse BOOTP option values
+	// See RFC2132, section 2. Also keep in mind
+	// "network byte order" is "big-endian order".
+	bootpEndian = binary.BigEndian
+)
+
+// ErrBootFileParamUnexpectedEnd is returned when option Bootfile param is
+// encoded wrongly. See https://www.ietf.org/rfc/rfc5970.txt (section 3.2).
+type ErrBootFileParamUnexpectedEnd struct {
+	Expected uint16
+	Left     uint16
+}
+
+// Error just implements interface "error"
+func (err *ErrBootFileParamUnexpectedEnd) Error() string {
+	return fmt.Sprintf("[bootfile param] unexpected end (expected: %d; left %d)", err.Expected, err.Left)
+}
+
+// OptBootFileParam implements the OptionBootfileParam option
+//
+// This module defines the OPT_BOOTFILE_PARAM structure.
+// https://www.ietf.org/rfc/rfc5970.txt (section 3.2)
+type OptBootFileParam []string
+
+var _ Option = OptBootFileParam(nil)
```

Comment:
What is this for ?

Reply:
To check if `OptBootFileParam` implements `Option` on build time. In this case it was actually copied from [`option_bootfileurl.go`](https://github.com/insomniacslk/dhcp/blob/3997b8a58c66233275ad9b26612ef733a63bd9d3/dhcpv6/option_bootfileurl.go#L13).